crates/forge/tests/code_quality.rs

```diff
@@ -0,0 +1,18 @@
+use camino::Utf8PathBuf;
+use packages_validation::check_and_lint;
+
+#[test]
+fn validate_snforge_std() {
+    let package_path: Utf8PathBuf = std::env::current_dir()
+        .expect("Failed to get current directory")
+        .ancestors()
+        .nth(2)
+        .expect("Failed to get grandparent directory")
+        .join("snforge_std")
+        .canonicalize()
+        .expect("Failed to canonicalize path")
+        .try_into()
+        .expect("Failed to convert to Utf8PathBuf");
```

Comment:
Why not just provide this path directly if the working directory of tests is stable?

```suggestion
    let package_path = Utf8PathBuf::from("../../snforge_std")
        .canonicalize()
        .unwrap()
        .try_into()
        .unwrap();
```